engine/src/parameters.rs

```diff
@@ -130,13 +130,48 @@ impl SubmitResult {
     }
 }
 
-/// Borsh-encoded parameters for the `call` function.
-#[derive(BorshSerialize, BorshDeserialize)]
+/// Borsh-encoded parameters for the engine `call` function.
+#[derive(BorshSerialize, BorshDeserialize, Debug, PartialEq, Eq, Clone)]
 pub struct FunctionCallArgs {
```

Comment:
Rename to `FunctionCallArgsV2`

Reply:
OK, fixed here https://github.com/aurora-is-near/aurora-engine/pull/351/commits/8e1306750eed1424c19f05d5ff40bfa61a6d0380